api/models/Ticket.js

```diff
@@ -24,7 +24,7 @@ const ticketSchema = new Schema({
 ticketSchema.statics.create = function (requestorId, description, tableNum, contactInfo) {
 	const ticket = new this({
 		requestorId,
-		timeFiled: Date.now,
+		timeFiled: new Date(),
```

Comment:
I would prefer using a number rather than an object when talking to the database, so `Date.now()` which will return a [Unix timestamp](https://en.wikipedia.org/wiki/Unix_time).

Reply:
yeah, that make more sense